currency_rate_update/services/__init__.py

```diff
@@ -18,3 +18,4 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+from . import update_service_HU_MNB
```

Comment:
Are you sure that this is necessary? The other services seemingly don't need to be imported here.